dvc/command/add.py

```diff
@@ -28,6 +32,12 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Recursively add each file under the directory.",
     )
+    add_parser.add_argument(
+        "--no-commit",
+        action="store_true",
+        default=False,
+        help="Don't save changes to cache.",
```

Comment:
changes -> files? something like "Do not put files into cache" or something like that?

Reply:
Agreed.